core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala

```diff
@@ -166,7 +166,7 @@ private class AsyncEventQueue(val name: String, conf: SparkConf, metrics: LiveLi
           val prevLastReportTimestamp = lastReportTimestamp
           lastReportTimestamp = System.currentTimeMillis()
           val previous = new java.util.Date(prevLastReportTimestamp)
-          logWarning(s"Dropped $droppedEvents events from $name since $previous.")
+          logWarning(s"Dropped $droppedCount events from $name since $previous.")
```

Comment:
I think the change is OK, since it tells us how many events dropped during the period `[System.currentTimeMillis(), lastReportTimestamp]`. 

BTW, I still have the same question with `lastReportTimestamp `(though, this is none of this pr's business):
When first time we log warning, `lastReportTimestamp ` should be `0` due to the default value and  without any initial. So, `previous` will be `1970/1/1 8:0:0 `.  And, it means we got several dropped events since `1970/1/1 8:0:0 `. I feel quite weird about it.

Reply:
Yeah that bothers me too!  Here is an example of that from the JIRA (boldness is mine):

> 18/04/04 09:55:57 WARN AsyncEventQueue: Dropped com.codahale.metrics.Counter@3348dc4a events from eventLog **since Wed Dec 31 18:00:00 CST 1969.**

At the very least it could be the time that the `AsyncEventQueue` is instantiated.  Simple as changing [L57](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala#L57)?

```scala
@volatile private var lastReportTimestamp = System.currentTimeMillis()
```

Reply:
Actually, I've tried init `lastReportTimestamp ` in `start()` in #20053. But, received disagreement from other contributors. I think the root cause of this problem is that we do not have a 'last time' for 'first time'. So, if we really want to fix it, we have to compromise on this.

Reply:
maybe the count should be a metrics, like https://github.com/phpisciuneri/spark/blob/3ad2f4298e290b3ce23533f04567bbd1fb096025/core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala#L72?